x-pack/plugins/alerting/server/config.ts

```diff
@@ -8,6 +8,21 @@
 import { schema, TypeOf } from '@kbn/config-schema';
 import { validateDurationSchema } from './lib';
 
+const ruleTypeSchema = schema.object({
+  id: schema.string(),
+  timeout: schema.maybe(schema.string({ validate: validateDurationSchema })),
+});
+
+const rulesSchema = schema.object({
```

Comment:
We will need to add this config to the docker allowlist and have a followup PR to add to the cloud allowlist.

Reply:
done, and a draft PR for cloud is also ready.